internal/config/config.go

```diff
@@ -54,6 +56,14 @@ func Init(name string, rootCmd cobra.Command, vip *viper.Viper, configChanged fu
 		vip.AddConfigPath("./")
 		vip.AddConfigPath("$HOME/")
 		vip.AddConfigPath("/etc/")
+
```

Comment:
nitpick: I wouldn’t add an extra line here and keept it with the previous one as it’s the same "stenza"